src/lib/viewers/doc/DocFirstPreloader.js

```diff
@@ -218,8 +217,11 @@ class DocFirstPreloader extends EventEmitter {
     }
 
     getPreloadImageRequestPromises(preloadUrlWithAuth, pages, pagedPreLoadUrlWithAuth) {
-        const promise1 = this.api.get(preloadUrlWithAuth, { type: 'blob' });
-        const promises = [promise1];
+        const firstPageUrl = !pagedPreLoadUrlWithAuth
+            ? preloadUrlWithAuth
+            : pagedPreLoadUrlWithAuth.replace(PAGED_URL_TEMPLATE_PAGE_NUMBER_HOLDER, '1.webp');
```

Comment:
Might be helpful to make `1.webp` a constant

Reply:
We have a lot of constants already and since that is only used in one place I felt like this would be ok.